util/test/testUtil.h

```diff
@@ -44,7 +45,7 @@ void valuecheck(const Eigen::DenseBase<DerivedA>& a, const Eigen::DenseBase<Deri
 
 void valuecheck(double a, double b, double tolerance = 1e-8)
 {
-  if (abs(a - b) > tolerance) {
+  if (std::abs(a - b) > tolerance) {
```

Comment:
yeah, let's do the drakeMexUtil thing. Is it OK to keep valuecheck in testUtil? I also need to get rid of eigenToMatlab and matlabToEigen from testUtil; I'll do that right now.